app/assets/stylesheets/layout/helper-classes.scss

```diff
@@ -341,6 +353,10 @@
   }
 }
 
+.charcoal-text {
+  color: $charcoal !important;
```

Comment:
Is this `!important` necessary here? The balance total on the DE receipt page works without.